litestar/config/app.py

```diff
@@ -207,3 +212,7 @@ def __post_init__(self) -> None:
         """
         if self.allowed_hosts and isinstance(self.allowed_hosts, list):
             self.allowed_hosts = AllowedHostsConfig(allowed_hosts=self.allowed_hosts)
+
+
+class ExperimentalFeatures(enum.StrEnum):
```

Comment:
Is an enum the best choice for this?

Reply:
I suppose you changed it to `class ExperimentalFeatures(str, enum.Enum):` to please CI on <3.11 versions ? what about keeping StrEnum and using the drop-in package (https://pypi.org/project/StrEnum/) for lower versions ? See https://blog.pecar.me/python-enum

Reply:
Hmm. Not sure, since the `(str, enum.Enum)` works just fine for now. If we have other use cases maybe?